### ISSUE TITLE
Error building example app

### ISSUE THREAD

**Issue by vegidio** [demchenkoalex/react-native-module-template#5]
I download the template, I ran `npm install` both in the project's root folder but also in the `example`folder and tried to build the example app right away, without changing anything, but the app doesn't run.

I keep getting the error below:

```
Error: Unable to resolve module `./index` from ``: 

None of these files exist:
  * index(.native|.ios.js|.native.js|.js|.ios.json|.native.json|.json|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx)
  * index/index(.native|.ios.js|.native.js|.js|.ios.json|.native.json|.json|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx)
```

**Comment by demchenkoalex** [comment]
Hi @vegidio can you please try with yarn instead of npm? You should run yarn in the root folder, example folder and npx pod-install in the example folder (unless you want to try Android only). I don't think that npm triggers the build script, that's why you don't see index.js

**Comment by vegidio** [comment]
Hi, I tried with yarn and followed the steps that you suggested, but it's still not working. I even recorded a video to show you and make sure that I didn't miss anything: https://www.youtube.com/watch?v=EUZRTX7De7U

Basically this is what I did:

1. Ran `yarn` in the project root folder; confirmed that the `lib` folder was successfully created.
2. Ran `yarn` in the `example` folder.
3. Ran `pod install` in the folder `example/ios`.
4. Tried to run the example project, but I keep getting the error that I reported before.
